spanner/spanner_samples.rb

```diff
@@ -1684,7 +1685,6 @@ def create_backup project_id:, instance_id:, database_id:, backup_id:
   instance = spanner.instance instance_id
   database = instance.database database_id
   expire_time = Time.now + 14 * 24 * 3600 # 14 days from now
-  version_time = client.execute("SELECT CURRENT_TIMESTAMP() as timestamp").rows.first[:timestamp]
 
   job = database.create_backup backup_id, expire_time, version_time: version_time
```

Comment:
@thiagotnunes I think here time is a string then need to parse it.

`Time.parse(version_time)`

Reply:
I don't think so, if we do the following it returns a `PreciseDate`

```
version_time = client.execute("SELECT CURRENT_TIMESTAMP() as timestamp").rows.first[:timestamp]
```

Reply:
Got it. Only concern is that running sample using command line it will not work. I think for this we should mention in documentation.

Reply:
In matter of fact, the `create_backup` can not be run through the command line at the moment. Check https://github.com/GoogleCloudPlatform/ruby-docs-samples/blob/master/spanner/spanner_samples.rb#L2062-L2074, where the `create_backup` requires a `backup_id` (https://github.com/GoogleCloudPlatform/ruby-docs-samples/blob/master/spanner/spanner_samples.rb#L1673). It will fail, because the `backup_id` is never passed in.

I tried adding another branch in the `if... else...` statement, but the cyclomatic complexity gets too high and the build fails. I think we should extract different files for the samples, this way it will be more in line with the other languages. However, I don't think it should be a part of this PR.

Reply:
Thanks for clarification.